content/articles/understanding-blockchain-as-a-service/index.md

```diff
@@ -0,0 +1,87 @@
+Make sure your BaaS provider has a track record of creating and Blockchain technology deployment and in a similar environment to the one you propose. If at all possible, inquire about qualifications or references from former clients.
+
+2. Commitment to Quality.
+Any enterprise application or software installation effort must prioritize quality. Blockchain is no exception. Inquire about the dedication to quality, process, and standards of the vendor.
+
+3. Assurance of safety.
+Examine the proposed implementation technique for any security issues. It's critical because even minor flaws can have serious consequences. When it comes to distributed ledgers, it is critical to plan for the most dependable results.
+4. Operation System Selection.
+Examine the previous installations of cloud-based services and the system they operate on that are similar to yours by the vendor. You must also guarantee that the new Blockchain installation works in tandem with your existing system.
+ 5. Usability.
+Ensure Blockchain-enabled systems and processes you're proposing are simple to use and implement. After all, you don't want a system that's difficult to use or navigate for your staff, since this could lower adoption rates and hurt your bottom line.
+ 6. Prices and Supporters.
+Every customer desires to pay the Cost of that which a BaaS provider provides. Examine alternatives of pricing as well as the deploying support and modalities carefully. Paying for that adds the biggest value to your company.
+
+###  Baas providers include:
+1. In 2015, Microsoft, in collaboration with ConsenSys, launched Ethereum blockchain-as-a-service on Azure (Microsoft).
+2. Amazon, which announced Amazon Managed Blockchain, which "makes it easier to construct and administer large blockchain networks" by leveraging open-source frameworks such as Ethereum or Hyperledger Fabric.
+3. Corda, a distributed financial ledger, was developed by R3, a collection of international financial institutions.
+4. Pay Stand is a corporation that specializes in sending and receiving payments between businesses.
+   
+    
+### Limitation of BaaS
+Consumers and organizations alike are becoming more open to using blockchain concepts. The presence of technical intricacies, the overhead operational association present to some building, configurations, operating, as well as maintaining a blockchain system, required infrastructure, on the other hand, frequently function as a barrier.
+
+For a price, BaaS hires an outside Baas company that provides the services to set up all of the required blockchain components as well as requirements. After these have been established, the client's sophisticated back-end processes are handled by the provider.
+
+Support services, such as bandwidth control, resource allocation, hosting requirements, and data security features, are often provided by the BaaS operator. Then allows the customer to what matters most: is blockchain's functioning.
```

Comment:
Can we control bandwidth or resource allocation or hosting even for public blockchain? 

Just curious.